Examples/Swift/CustomViewController.swift

```diff
@@ -96,22 +96,22 @@ class CustomViewController: UIViewController, MGLMapViewDelegate, AVSpeechSynthe
     func updateRouteProgress(routeProgress: RouteProgress) {
         guard let step = routeProgress.currentLegProgress.upComingStep else { return }
 
-        if let direction = step.maneuverDirection {
-            switch direction {
-            case .slightRight:
-                self.arrowView.text = "↗️"
-            case .sharpRight, .right:
-                self.arrowView.text = "➡️"
-            case .slightLeft:
-                self.arrowView.text = "↖️"
-            case .sharpLeft, .left:
-                self.arrowView.text = "⬅️"
-            case .uTurn:
-                self.arrowView.text = "⤵️"
-            default:
-                self.arrowView.text = "⬆️"
-            }
+        let direction = step.maneuverDirection
```

Comment:
This variable can be removed and you can just switch on `step.maneuverDirection `